types/compilation/compilation.interfaces.ts

```diff
@@ -48,6 +48,7 @@ export type CompilationResult = {
     compilationOptions?: string[];
     downloads?: BuildEnvDownloadInfo[];
     gccDumpOutput?: any;
+    languageId: string;
```

Comment:
My suggestion is to make this:
```suggestion
    languageId?: string;
```